index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Revera Clone</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles.css"/>
+</head>
+<body>
+    <!-- NAVIGATION BAR-->
+    <nav class="navbar navbar-expand-sm navbar-light bg-light ">
+        <div class="container">
+            <a class="navbar-brand" href="#">REVERA</a>
+            <!-- <button class="navbar-toggler" data-toggle="collapse" data-target="#navbarNav">
+                <span class="navbar-toggler-icon"></span>
+            </button> -->
+            <div class="collapse navbar-collapse" id="navbarNav">
+                <ul class="navbar-nav ml-auto">
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Home</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Sample Page</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Portfolio</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Blog</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Parent Page</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Contact</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <!-- CAROUSEL -->
+    <div class="container-fluid">
+        <div class="row">
+            <div class="carousel-inner">
+                <div id="front slider" class="carousel slide" data-ride="carousel">
```

Comment:
You can get the carousels control arrows to work if you make sure that their `href` and `id` attributes are the same
```
<div id="slider3" class="carousel slide" data-ride="carousel">
...
<a href="#slider3" class="carousel-control-prev" data-slide="prev">
      <span class="carousel-control-prev-icon"></span>
   </a>

  <a href="#slider3" class="carousel-control-next" data-slide="next">
       <span class="carousel-control-next-icon"></span>
  </a>
</div>